WebAPI/Core/Services/AuthServices.cs

```diff
@@ -15,14 +15,35 @@ public class AuthServices
     {
         private readonly IOptions<AuthOptions> _authOptions;
         private readonly IAuthRepository _authRepository;
+        private readonly HashingService _hashingService;
 
-        public AuthServices(IOptions<AuthOptions> authOptions, IAuthRepository authRepository)
-            => (this._authOptions, this._authRepository) = (authOptions, authRepository);
+        public AuthServices(IOptions<AuthOptions> authOptions, IAuthRepository authRepository, HashingService hashingService)
+            => (this._authOptions, this._authRepository, this._hashingService) = (authOptions, authRepository, hashingService);
```

Comment:
Use usual indentation instep of an arrow function when you have more than 2 parameters.